### PR TITLE
Issue #1: Add SRS document for the website 📝

### LINKED ISSUE #1: Create SRS Document
A Software Requirements Specification document needs to be prepared so that developers are up to date with the current progress and the plan for the website.

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
Resolves #1 

### To Reviewers:

Feel free to add suggestions and commits, don't modify existing commit, new commit should be added if some reviewer wants to push changes by themselves.

- [x] Commit Signed

### CODE REVIEW

File: SRS-Document/SRS-Document.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+There are no constrains as of now.
+
+### 2.4 User Characteristics
+
+All user classes are crucial to satisfy but broadly there are two user classes:
+- Bloggers or folks who will write blog posts.
+- Readers of those blogs.
+  
+Writing a blog should be a smooth experince with interactive UI and should support markdown.
```
Comment (AshAman999) [inline SRS-Document/SRS-Document.md:85]:
```suggestion
Writing a blog should be a smooth experience with interactive UI and should support markdown.
```

**Review by AshAman999 — APPROVED** [review]
Looks good to go for the first iteration ,
minor  comments, rest LGTM :)

File: SRS-Document/SRS-Document.md (hunk 1 above)
Comment (Vyom-Yadav) [reply SRS-Document/SRS-Document.md:85]: @AshAman999 Let's merge #4 first, that will help us in the future too, this PR will act as a test case for it.

**Comment by Rahulkhinchi03** [comment]
@Vyom-Yadav I believe we should add one DEVELOPER OR CORE TEAM OR MEMBERS Section in the SRS to make it official.

Also, do you have any idea of how we can proceed with licensing our community as we will need some terms and conditions to future-proof our community to avoid conflicts in the future?

File: SRS-Document/SRS-Document.md (hunk 1 above)
Comment (Rahulkhinchi03) [reply SRS-Document/SRS-Document.md:85]: @Vyom-Yadav @AshAman999 let's merge it, whatever the changes are (related to spelling and small errors), we will resolve them by making good first issues to improve grammar.

Can we merge?

Comment (Rahulkhinchi03) [reply SRS-Document/SRS-Document.md:85]: One thing needed that we will be make a new issue for:

Adding one DEVELOPER OR CORE TEAM OR MEMBERS Section in the SRS to make it official.

**Pull request merged**